tutorials/receiving-money.md

```diff
@@ -0,0 +1,275 @@
+
+The `amount` values that you pass to Paystack must be in kobo. To convert a Naira value to kobo, multiply by 100. So if you wanted to charge N100, the amount value to send will be 10000.
+
+###Adding Extra Information to the Payment Request
+
+Sometimes you want to pass extra information to your transaction. The initialization request body accepts a `metadata` object in which you can add other information you want. You can pass the object directly into the `metadata`. You can also use `custom_fields` array within the metadata. Custom fields is an array of objects. The values you pass in the `custom_fields` array will displayed in the transaction details on the Paystack Dashboard and on the customer's receipt.
+
+A transaction initialization object with `metadata` will look like this:
+
+```JSON
+{
+  email: "me@you.us",
+  amount: 100000,
+  metadata: {
+    account_id: "i923d923",
+    reg_number: "101290",
+    custom_fields: [
+      {
+        display_name: "Favourite Movie",
+        variable_name: "favourite_movie",
+        value: "Pirates of the Carribeans"
+      },
+      {
+        display_name: "Worst Movie",
+        variable_name: "worst_movie",
+        value: "La la Land"
+      }
+    ]
+  }
+}
+```
+
+####Adding Custom Filters
+
+Custom Filters allow you to set some preferences on the checkout form. For example, you can decide to receive only certain card types on the **Pay With Card** channel. If you were building an application that requires recurring debit, you can choose to allow customers enter only cards that support recurring debit. For **Pay With Bank**, you can choose to accept only certain banks by passing thier bank codes ([here is how to get list of banks and their bank codes][https://developers.paystack.co/reference#list-banks]).
+
+These settings are passed in the **custom_filters** object passed in the **metadata**. Using custom filters, your **metadata** will look something like this:
+
+```
+{
+    email: 'your@email.com',
+    amount: 10000,
+    metadata: {
+        custom_filters: {
+            recurring: true,	//only accept cards that support recurring debit
+            banks: ['057', '100'],  // bank codes to accept in Pay with Bank Channel
+         	card_brands: ["mastercard", "visa"] //only accept mastercard and visa cards
+        }
+    }
+}
+```
+
+ 
+
+### Response Structure
+
+A typical Paystack response object (response) contains:
+
+- status
+- message
+- data
+
+**Status** is true or false depending on if the API call is was processed or not. If you something was wrong with your request (no authorization object, no/wrong secret key, invalid/duplicate transaction reference) the status will be false. The status is NOT the status of the transaction, it is the status of the API call.
```

Comment:
**Status** is true or false depending on if the API call was processed or not. If something was wrong with your request (no authorization object, no/wrong secret key, invalid/duplicate transaction reference) the status will be false. The status is NOT the status of the transaction, it is the status of the API call.